apps/mark-scan/backend/package.json

```diff
@@ -86,7 +87,8 @@
     "lint-staged": "11.0.0",
     "nodemon": "^2.0.20",
     "sort-package-json": "^1.50.0",
-    "ts-jest": "29.1.1"
+    "ts-jest": "29.1.1",
+    "wait-for-expect": "^3.0.2"
```

Comment:
nit: there is a `backendWaitFor` helper in test-utils

Reply:
[d8ecba4](https://github.com/votingworks/vxsuite/pull/4628/commits/d8ecba46a7037fb31710b150a56c37b4a61dfd4b)